mobile-academy/src/main/resources/ma-imi.properties

```diff
@@ -0,0 +1,10 @@
+# TODO: replace nmssenderid with a real production sender id
+imi.sms.sender.id = nmsfoobar
+
+# TODO: replace localhost with a real service url
+imi.sms.notification.url = http://localhost/smsmessaging/v1/outbound/nmssenderid/requests
+
+# TODO: figure out where to get the sms content from (already emailed bbc)
+imi.sms.course.completion.message = This is a foobar message that needs to be replaced
```

Comment:
I have two threads going with IMI and BBC to figure out where/how we could get the sms content from. Tracking in #278 and will update once we have a resolution